ProcessMessage/__tests__/handler.test.ts

```diff
@@ -402,22 +423,24 @@ describe("getprocessMessageHandler", () => {
+    ${"message sender special service does not exists in user service preference and Activation is PENDING near grace period"} | ${"SENDER_BLOCKED"}        | ${O.some(aRetrievedProfileWithManualPreferences)}                                                             | ${O.none}                                                        | ${O.some({ ...aPendingActivation, _ts: subSeconds(ExecutionDateContext, DEFAULT_PENDING_ACTIVATION_GRACE_PERIOD_SECONDS).getTime() })} | ${aCreatedMessageEvent} | ${aSpecialMessageData}
+    ${"message sender special service exists in user service preference and Activation is INACTIVE"}                           | ${"SENDER_BLOCKED"}        | ${O.some(aRetrievedProfileWithManualPreferences)}                                                             | ${O.some(anEnabledServicePreference)}                            | ${O.some(aDisabledActivation)}                                                                                                         | ${aCreatedMessageEvent} | ${aSpecialMessageData}
+    ${"message sender special service exists in user service preference and Activation not exists"}                            | ${"SENDER_BLOCKED"}        | ${O.some(aRetrievedProfileWithManualPreferences)}                                                             | ${O.some(anEnabledServicePreference)}                            | ${O.none}                                                                                                                              | ${aCreatedMessageEvent} | ${aSpecialMessageData}
+    ${"message sender special service exists in user service preference and Activation is PENDING"}                            | ${"SENDER_BLOCKED"}        | ${O.some(aRetrievedProfileWithManualPreferences)}                                                             | ${O.some(anEnabledServicePreference)}                            | ${O.some(aPendingActivation)}                                                                                                          | ${aCreatedMessageEvent} | ${aSpecialMessageData}
+    ${"message sender special service exists in user service preference and Activation is PENDING near grace period"}          | ${"SENDER_BLOCKED"}        | ${O.some(aRetrievedProfileWithManualPreferences)}                                                             | ${O.some(anEnabledServicePreference)}                            | ${O.some({ ...aPendingActivation, _ts: subSeconds(ExecutionDateContext, DEFAULT_PENDING_ACTIVATION_GRACE_PERIOD_SECONDS).getTime() })} | ${aCreatedMessageEvent} | ${aSpecialMessageData}
```

Comment:
how can this scenario be possible?

Reply:
Ok, I thought that service preferences were bounded to special service activation, my bad